lightning/src/ln/channel.rs

```diff
@@ -6550,8 +6565,23 @@ impl<'a, K: Deref> ReadableArgs<(&'a K, u32)> for Channel<<K::Target as KeysInte
 			(21, outbound_scid_alias, option),
 			(23, channel_ready_event_emitted, option),
 			(25, user_id_high_opt, option),
+			(27, channel_keys_id, option),
 		});
 
+		let (channel_keys_id, holder_signer) = if let Some(channel_keys_id) = channel_keys_id {
```

Comment:
is this if clause necessary yet? For the time being, we're always gonna be reading the channel signer from disk, so I think we can probably delay the introduction of this clause until we actually stop writing it and have a version change with an upgrade